docker/optimizer/build.sh

```diff
@@ -0,0 +1,3 @@
+ docker build -t optimizer:0.6.0  .
+ docker tag optimizer:0.6.0    registry.cn-hangzhou.aliyuncs.com/udh/amoro-optimizer:0.6.0
+ docker push  registry.cn-hangzhou.aliyuncs.com/udh/amoro-optimizer:0.6.0
```

Comment:
This is not a good docker build.sh. Version, registry, and other information should not be hardcoded in the build.sh file.

Please refer to the `docker/build.sh` file.